### DISCUSSION TITLE
Cant seem to get Mac os 12+ option on UTM

### DISCUSSION THREAD

**Discussion by Gamerboo0fficial** [utmapp/UTM#4877] [UTM Help (macOS)] [answered]
when clicking the "+" and virtualisation option there is only windows os and Linux there is no mac os option please help

**Comment by osy** [comment, marked as answer]
Intel does not support macOS virtualization on UTM.

**Reply by Gamerboo0fficial** [reply]
so is there other softwares that i can simulate/emulate mac os? or at least some of the older version like lion or El captain

**Reply by osy** [reply]
Parallels or vmware

**Reply by Gamerboo0fficial** [reply]
i have tryed those two but they dont work i made sure my system is compatible is there a way to upgrade from mac os 9.2.1 prebuilt to at least mac os X? 

this was my main goal was to try to find a way to update mac os 9.2.1 to mac os x 

this youtube video was the reason i started https://www.youtube.com/watch?v=l1vW0cOAfro&ab_channel=RossDarker (skip to 8:20)

thanks if you can help
(the only reason i ask is because UTM uses QEMU and the person in the video was useing QEMU)

Thanks for your time <3

**Reply by Unknown1297471** [reply]
a 2 year-late answer: use OCLP.

**Reply by kaeziinamphet12-ux** [reply]
Install macOS Monterey zip utm
